src/store/reducers/trace.ts

```diff
@@ -0,0 +1,28 @@
+import type {BaseQueryFn, EndpointBuilder} from '@reduxjs/toolkit/query';
+
+import {api} from './api';
+
+interface CheckTraceParams {
+    url?: string;
+}
+
+function endpoints(build: EndpointBuilder<BaseQueryFn, string, string>) {
+    return {
+        checkTrace: build.query<{}, CheckTraceParams>({
+            queryFn: async ({url}, {signal}) => {
```

Comment:
```
checkTrace: build.query({
        queryFn: async ({url}: CheckTraceParams, {signal}) => {
```

This way it will use your response type and you won't need to add it by yourself